frontend/src/components/eam/eam.tsx

```diff
@@ -39,6 +39,25 @@ const EAM: FC = () => {
     dispatch(eamActions.deleteGroup(id));
   };
 
+  const handleWorkersReload = (): void => {
+    dispatch(
+      eamActions.loadWorkers({
+        tenantId: tenantId as string,
+        from: 0,
+        count: 5,
+      }),
+    );
+  };
+  const handleGroupsReload = (): void => {
+    dispatch(
+      eamActions.loadGroups({
+        tenantId: tenantId as string,
+        from: 0,
+        count: 5,
+      }),
+    );
+  };
+
```

Comment:
it should've been merged... 
IDK why is it green if it already exists on develop